File: abes/capegemini/mywebprj/WebContent/es6_example/sum.ts
function sum(n1:number, n2:number):void{
     console.log(`sum of ${n1} and ${n2} is ${n1+n2}`)
}
function multiplication(n1:number, n2:number):number{
return n1*n2;
}

function arraySum(n:number[]):void{
   let sum:number=0;
    for(var k=0;k<n.length;k++){
      sum=sum+n[k];
    }
    console.log('sum is '+sum)
}

sum(11,33);

let res=multiplication(4,3);
console.log('multiplication is '+res);
arraySum([1,2,3,4,5]);

File: junit_exception_scanner/CoreJava_Scanner/src/main/java/net/rctay/testing/sample/Sum.java
package net.rctay.testing.sample;

import java.util.Scanner;

public class Sum {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		// by convention, number of cases - throw that away here
		sc.nextInt();

		int result = 0;
		int n = sc.nextInt();
		while (n-- > 0)
			result += sc.nextInt();

		System.out.println(result);
	}
}
